packages/react-server-cli/src/index.js

```diff
@@ -0,0 +1,52 @@
+import parseCliArgs from "./parseCliArgs"
+
+// weirdly, we parse the args twice. the first time we are just looking for --production, which
+// affects the default values for the other args.
+const isProduction = parseCliArgs(false).production || (process.env.NODE_ENV === "production"); //eslint-disable-line no-process-env
+// now if production was sent in on the command line, let's set NODE_ENV if it's unset.
+if (isProduction && !process.env.NODE_ENV) { //eslint-disable-line no-process-env
+	process.env.NODE_ENV = "production"; //eslint-disable-line no-process-env
+}
+const argv = parseCliArgs(isProduction);
+
+// TODO: do we need a post-processor for logger here?
+// these require calls are after the argument parsing because we want to set NODE_ENV
+// before they get loaded.
+const logging = require("react-server").logging,
+	logger = logging.getLogger({name: "react-server-cli/index.js", color: {server: 9}}),
```

Comment:
Can this be made to use `react-server-gulp-module-tagger` so we don't have a hard-coded `__LOGGER__` object here?